internal/impl/pure/bloblang_objects.go

```diff
@@ -125,6 +125,9 @@ If a key within a nested path does not exist then it is ignored.`).
 			}
```

Comment:
I'd say it's probably more intuitive to return an error here when no arguments are present as this'd indicate misuse of the method, maybe someone assuming this method creates a zip archive.

Reply:
Good call! I was on the fence about it, but I agree that returning an error is a better idea. Fixed 🎉